main.go

```diff
@@ -71,6 +71,35 @@ func main() {
 		}
 	}
 
+	// The goroutine will continuously check bitcoind usage and auto close bitcoind if the client isn't active.
+	go func(checkInterval time.Duration) {
+		for {
+			autoCloseTime := time.Now().Add(time.Minute * time.Duration(-viper.GetInt("bitcoind_ctl.suspending_duration"))).Unix()
+			if controller.LastActiveTime.Unix() < autoCloseTime {
+				status, err := controller.getBitcoindStatus()
+				if err != nil {
+					log.WithError(err).Error("fail to auto check bitcoind status")
+					time.Sleep(checkInterval)
+					continue
+				}
+				statusNotready, _ := json.RawMessage(`{"error":"bitcoind process is not ready"}`).MarshalJSON()
+				statusDown, _ := json.RawMessage(`{"error":"bitcoind is stopped"}`).MarshalJSON()
+				currentStatus, _ := status["responseBody"].(json.RawMessage).MarshalJSON()
```

Comment:
You can introduce a structure to prevent from doing type casting. For example, 
```
type BitcoindCtlResponse struct {
	Status		int
	ResponseBody	string
}
```